File: App_Resources/Android/src/main/java/fr/helfrich/kt/DatadomeHelper.kt
package fr.helfrich.kt

class DatadomeHelper(val rootUrl: String) {
    var cookieName = ""
    //private lateinit var dataDomeSDK: SDKBase
    //private DataDomeSDK.Builder dataDomeSDK;

    fun showRootURL() { 
        println("Kotlin : rootUrl = ${rootUrl}")
    }

    // Bloc d'initialisation
    init {
        println("Init DatadomeHelper")

        /*
        dataDomeSDK = SDKBase
            .Builder()
            .listener(captchaListener)
            .context(this)
            .backBehavior(BackBehavior.GO_BACK)
            .build()

        val cookie = dataDomeSDK.cookie("https://advango.fr")
        if (!cookie.isNullOrBlank()) {
            cookieName = cookie.name
        }
        */
    }
}

File: App_Resources/iOS/src/DatadomeHelper.swift
import UIKit

import DataDomeSDK

@objcMembers
class DatadomeHelper: NSObject {
    var rootUrl: String
    var cookieName: String?

    init(rootUrl: String) {
        self.rootUrl = rootUrl

        if let cookie = DataDomeSDK.getCookie() {
             self.cookieName = cookie.name
        }
    }

    func showRootURL() {
        print("[DatadomeHelper Swift] rootUrl = \(self.rootUrl)")
    }
}